manifests/editors/atom.pp

```diff
@@ -52,6 +52,24 @@
         require => Package['Atom'],
       }
     }
+    'Debian': {
+      validate_string($url)
+      $apm_require = Package['atom']
+
+      package { 'atom':
+        ensure   => $ensure,
+        source   => '/tmp/atom-amd64.deb',
+        provider => 'dpkg',
+      }
+
+      file { '/tmp/atom-amd64.deb':
+        source => 'https://atom.io/download/deb',
```

Comment:
On Puppet 4.8 this doesn't work for some reason. Despite the [Puppet documentation](https://puppet.com/docs/puppet/4.8/types/file.html#file-attribute-source) saying it should work. I had to fall back to using `archive` for downloading the .deb file. Not sure why. I remember trying hard before giving up.

Reply:
But then again: We're using _this_ module, _this_ branch in production. And it _works_. :smirk: :astonished: 

I think the world is just flat, really.

Reply:
lol, it works for us in our environment.